Clean up utils directory

AlysonStahl-NOAA opened the pull request:
Fixes #712

AlysonStahl-NOAA committed:
consolidated files for degrib2

AlysonStahl-NOAA committed:
temporarily disable test

AlysonStahl-NOAA committed:
combined some files for cnvgrib

edwardhartnett committed:
Merge branch 'develop' into as_utils2

edwardhartnett commented:
@AlysonStahl-NOAA sorry, I didn't notice this was still just a draft. Looks good though!

AlysonStahl-NOAA committed:
merge mkfldsep into tocgrib

AlysonStahl-NOAA committed:
subroutines for degrib2 used separately in test, combined into their own file

AlysonStahl-NOAA commented:
I think this should be done now. 

The file for cnvgrib would be too long if I moved all of it's subroutines into it, but I managed to condense it down to 4 total files.

The subroutines used by degrib2 are used by a test so I moved them all into one separate file. 

makwmo.f90 is used by two utilities, so I kept that separate as well.
